packages/luvi/build.sh

```diff
@@ -2,21 +2,20 @@ TERMUX_PKG_HOMEPAGE=https://luvit.io
 TERMUX_PKG_DESCRIPTION="A project in-between luv and luvit."
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Komo @cattokomo"
-TERMUX_PKG_VERSION=20240216
-TERMUX_PKG_REVISION=2
-_commit=b85da58f2549a519486a7296f5b836a8f6a64880
-_version=2.14.0
+TERMUX_PKG_VERSION=20240723
+_commit=4e92094134161601a841d2ff5c8b0759fcc743b1
+_version=2.14.0+g${_commit::7}
 TERMUX_PKG_GIT_BRANCH=master
 TERMUX_PKG_SRCURL=git+https://github.com/luvit/luvi.git
-TERMUX_PKG_DEPENDS="pcre, openssl, luv, lua51-lpeg, libluajit"
```

Comment:
Wait, no more lpeg?

Reply:
I think it's better to leave it like that, since it just adds another problem when bumping. Let the build system handle lpeg dependency.

Reply:
So... Lpeg is vendored in?

Reply:
Yep.

Reply:
Ah okay